tools/tool-migration-utility/src/main/java/org/sagebionetworks/tool/migration/v3/SynapseClientFactoryImpl.java

```diff
@@ -42,7 +45,10 @@ public SynapseAdminClientImpl createNewDestinationClient() throws SynapseExcepti
 	 * @throws SynapseException 
 	 */
 	private static SynapseAdminClientImpl createNewConnection(SynapseConnectionInfo info) throws SynapseException{
-		SynapseAdminClientImpl synapse = new SynapseAdminClientImpl();
+		HttpClientProvider provider = new HttpClientProviderImpl();
+		provider.setGlobalConnectionTimeout(1000*60); 		// 	1 	min

```

Comment:
I thought this was the time to setup/open the connection (the default is
5s).

On Tue, Jun 14, 2016 at 11:01 AM, John Hill <notifications@github.com>
wrote:

> In
> tools/tool-migration-utility/src/main/java/org/sagebionetworks/tool/migration/v3/SynapseClientFactoryImpl.java
> <https://github.com/Sage-Bionetworks/Synapse-Repository-Services/pull/2460#discussion_r67023328>
> :
>
> > @@ -42,7 +45,10 @@ public SynapseAdminClientImpl createNewDestinationClient() throws SynapseExcepti
> >  	 * @throws SynapseException
> >  	 */
> >  	private static SynapseAdminClientImpl createNewConnection(SynapseConnectionInfo info) throws SynapseException{
> > -		SynapseAdminClientImpl synapse = new SynapseAdminClientImpl();
> > +		HttpClientProvider provider = new HttpClientProviderImpl();
> > +		provider.setGlobalConnectionTimeout(1000*60); 		// 	1 	min
>
> Why only one minute for the global timeout? Some of the migration calls
> can take much longer (5+mins).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Sage-Bionetworks/Synapse-Repository-Services/pull/2460/files/ebdf8b5ee2fd91c9223ac9bae498d97d7581a873#r67023328>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABrzD-5Ca9G7KUUgyd7Eg3aikz9_81M0ks5qLux_gaJpZM4I02hN>
> .
>